### DISCUSSION TITLE
General Informations exchanges for self hosted version of Supabase

### DISCUSSION THREAD

**Discussion by anasfik** [supabase/supabase#12954] [General]
Hello, I discovered Supabase a while, the cloud service is just great and unique.
However, working with the self-hosted version let to a lot of confusing such as regarding the existence of features, settings, different databases, and authentication behaviors that I love to discuss here.

**Comment by cristian-milea** [comment]
self hosted version seems like a dead-end also to me. not enough information to get it running by yourself. did you get it running?

**Reply by anasfik** [reply]
the self-hosted topic was really hard to cover, Supabase has still limited in comparison to Firebase and is not a full competitor to Firebase, but I can't ignore that it comes with great features such as the use of the Postgres DB..., However this still good when using the cloud platform of Supabase.

when it comes to the self-hosted version, it is like a limited version of the limited cloud version Superbase cloud, I was really disappointed a little when I try it that I spend almost 1 week researching and deploying with many ways thinking that I do something wrong:
 - There are no cloud/edge functions features (the most disappointing thing)
 - There is no support for creating multiple projects.
 - There is no direct settings dashboard in the studio (thanks to this actually, I've been put in a situation when I've been forced to deal with docker and docker-compose to change settings, and deploy snapshots...)
 - There is no clear documentation for it, I mean full documentation that covers everything.
 - There is a lack of other features such as backups, reports, and logs...

but even this, I still recommend it for using its database, authentication, and storage  APIs, and getting into the pain of self-hosting will make you learn new things, I do believe in the potential of this project.

and for me, the edge/cloud functions lack in the self-hosted is the most thing that I want to see in the self-hosted version.  

if you have any questions about it, please ask me, I might help you!

**Comment by squallsama** [comment]
I was able to set-up self hosted Supabase with nginx reverse proxy, http basic auth for studio, ssl and all of it inside of a docker compose. Most disappointing thing - lack of cloud functions and because of that I'll use Appwrite cloud functions among with supabase db and auth

**Comment by soedirgo** [comment]
As of https://github.com/supabase/supabase/pull/14654, supabase-js should work OOtB with Edge Functions on the [self-hosted Docker setup](https://supabase.com/docs/guides/self-hosting/docker#setting-up-edge-functions). Let us know if you're having issues setting it up.

**Comment by CarlosBranty** [comment]
Hi, I tried to deploy selfhosted supabase on a vps following some YouTube tutorials, but when I finish and try to create a new user from studio I get an error, which tells me that fetch cannot be done also after deleting everything and installing again supabase storage does not work Additionally, for authentication do I need to do some more configuration? I would greatly appreciate your help.
